signer/src/config/mod.rs

```diff
@@ -167,12 +167,9 @@ impl Validatable for BitcoinConfig {
             }
         }
 
-        if self.chain_tip_polling_interval.as_secs()
-            > MAX_BITCOIN_CHAIN_TIP_POLLING_INTERVAL_SECONDS
-        {
+        if self.chain_tip_polling_interval == std::time::Duration::ZERO {
```

Comment:
Are we sure we want 0 as minimum? If minimum is zero why we need this constraint at all? Like, since minimum is 0 I can put it to 1 nanosecond and it will make no difference compared to 0. Also, since we parse this interval from number of seconds in config file, tecnical minimum is 1 second.
```suggestion
        if self.chain_tip_polling_interval <= std::time::Duration::from_secs(1) {
```

Reply:
We only have seconds resolution for that duration, https://github.com/stacks-sbtc/sbtc/blob/ecc0402102fcb2fc3b377547bb195a74a72672b0/signer/src/config/mod.rs#L141-L142